modules/microshift-install-rpm-preparing.adoc

```diff
@@ -6,30 +6,27 @@
 [id="microshift-install-rpm-preparing_{context}"]
 = Preparing to install {product-title} from an RPM package
 
-Before installing {product-title} from an RPM package, you must configure your {op-system} machine to have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
-
-{product-title} uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider for storing PVs. LVMS relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG with unused space in which LVMS can create the LVM logical volumes (LVs) for your workload's PVs.
-
-To configure a volume group (VG) that allows LVMS to create the LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
+Configure your {op-system} machine to have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
 
 .Prerequisites
 
 * The system requirements for installing {product-title} have been met.
 * You have root user access to your machine.
+* You have configured your LVM VG with the capacity needed for the PVs of your workload.
 
 .Procedure
 
 . In the graphical installer under *Storage Configuration*, select *Custom* -> *Done* to open the dialog for configuring partitions and volumes.
```

Comment:
```suggestion
. In the graphical installer under *Installation Destination* in the *Storage Configuration* subsection, select *Custom* -> *Done* to open the dialog for configuring partitions and volumes. This will take you to a window titled *Manual Partitioning*.
```